projects/igniteui-angular/src/lib/test-utils/grid-samples.spec.ts

```diff
@@ -1591,13 +1591,32 @@ export class DynamicColumnsComponent extends GridWithSizeComponent {
             <igx-checkbox [checked]="headContext.totalCount === headContext.selectedCount"
                 (click)="onHeaderCheckboxClick($event, headContext)"></igx-checkbox>
         </ng-template>
-    </igx-grid>`
+    </igx-grid>
+    <ng-template #customRow igxRowSelector let-rowContext>
+            <span class="rowNumber">CUSTOM SELECTOR: {{ rowContext.index }}</span>
+    </ng-template>
+    <ng-template #customHeader igxHeadSelector let-headContext>
+        <span>CUSTOM HEADER SELECTOR</span>
+    </ng-template>
+    <ng-template #customGroupRow igxHeadSelector let-headContext>
```

Comment:
This should be `igxGroupByRowSelector` but you dont use the context so there should not be any difference.